### ISSUE TITLE
(M2C) (SPC) run M2C existence check on prod (blocked by #268  )

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/preservation_catalog#123]
blocked by #122 and other M2C existence check tickets

**Comment by ndushay** [comment]
please put data in the wiki

**Comment by ndushay** [comment]
blocked by #209 (seed_catalog run in prod)

**Comment by ndushay** [comment]
duplicate of #398.  closing
